.buildkite/pipeline.yml

```diff
@@ -62,3 +62,15 @@ steps:
           - step: "int-test"
             allow_failure: true
     depends_on: "check"
+
+  - label: "Publish docker image"
+    key: "publish"
+    steps:
+      - label: "Build docker image"
+        command: ".buildkite/scripts/build_image.sh"
+        if: "!build.pull_request"
```

Comment:
:+1:

Reply:
I changed the condition to:
if: build.env("BUILDKITE_PULL_REQUEST") == null